Events/views.py

```diff
@@ -1,3 +1,72 @@
-from django.shortcuts import render
+from django.shortcuts import render, get_object_or_404, redirect
+from .models import Event, Participant
+from django.http import HttpResponse, HttpResponseRedirect, Http404
+from django.contrib.auth import get_user_model
+from django.utils import timezone
+from .forms import EventForm, ParticipantForm
 
-# Create your views here.
+
+def event_list(request):
+    events = Event.objects.all()
+    context = {
+        "events": events
+    }
+
+    return render(request, 'Events/event_list.html', context)
+
+
+def event_create(request):
+    if request.user.is_authenticated:
```

Comment:
Authenticated users only should be able to add events. Change the if statement. Also, check initially if the request is POST or GET to decide whether a form is being submitted or a new event is being created.